unity/Assets/Tests/EditMode/Logic/CoordinateTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Hexxle.CoordinateSystem;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace Hexxle.Tests.Logic
+{
+    public class CoordinateTests
+    {
+        [Test]
+        public void Conversion_Hexxle91()
```

Comment:
Do we need the taskname in the function name?

Reply:
Since we plan on specifically testing for a bug with values that made that bug appear, I thought it would be a good idea to reference that.

In the end, it's not really something that matters. If you have a suggestion on how to name this test, I'm very open to changing it.